README.md

```diff
@@ -72,16 +65,28 @@ by this point.
     cd NAB
     (sudo) pip install -r requirements.txt
 
-This will install the requirements, such as pandas and simplejson.  
+This will install the additional required modules pandas and simplejson.  
 
 
 ### Usage
 
+##### Run NAB
+
     cd /path/to/nab
-    python setup.py develop (for now)
     python run.py
 
-This will produce results files for the Numenta anomaly detection method, as well as baseline results using methods from the [Etsy Skyline](https://github.com/etsy/skyline) anomaly detection library, and a random detector. This will also pass those results files to the analyze_results.py script to generate final scores.
+This will produce results files for the desired anomaly detection methods; the user is prompted which algorithm(s) to run. Included in the repo are the Numenta anomaly detection method, as well as methods from the [Etsy Skyline](https://github.com/etsy/skyline) anomaly detection library, and a random detector. This will also pass those results files to the analyze_results.py script to generate final NAB scores.
+
+##### Run your detector
+
+To run your own detector (named 'DUT' for detector under test), first create a folder in the nab detectors directory:
```

Comment:
This statement is actually misleading as there are a couple of different ways to run your detector including a variation that is much simpler than what is specified here. I think we should take the details out of the README and instead point them to the Word document.